app/templates/components/conversations/conversation-part-closed.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="conversation-part-closed">
```

Comment:
This `div` is not necessary since the component itself creates a `div` with this class (using the `classNames` you defined).